### ISSUE TITLE
Use with Nuxt3 ?

### ISSUE THREAD

**Issue by estige** [n4j1Br4ch1D/postcss-obfuscator#16]
This is compatible?

**Comment by monckey100** [comment]
No, doesn't work with Nuxt 2 either, spent a lot of time trying to get this to work only to find out that it causes webpack builds to fail because it initializes midway through a build instead of at the very end.
